src/gui/SearchDialog.cpp

```diff
@@ -667,6 +672,73 @@ void SearchDialog::setSimpleStyle()
 	ui->coordinateSystemComboBox->setVisible(false);
 }
 
+void SearchDialog::setCenterOfScreenCoordinates()
+{
+	StelCore *core = StelApp::getInstance().getCore();
+	const auto projector = core->getProjection(StelCore::FrameJ2000, StelCore::RefractionMode::RefractionOff);
+	Vec2i centerScreen(projector->getViewportPosX() + projector->getViewportWidth() / 2,
```

Comment:
How does this interact with `StelProjector::viewportCenterOffset`?

Reply:
No, this tool doesn’t interact with `StelProjector::viewportCenterOffset`